client/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@libero-reviewer/client",
+  "name": "@libero-reviewer/clients",
```

Comment:
What's with the pluralisation here?

Reply:
That's just some junk data testing if the script picks up changes correctly.